### PR TITLE
fix(std): Overflow in `truncate`

### LINKED ISSUE #264: `truncate` has overflow problems
`thread 'conformance_ruby::standard_filter_test::test_truncate' panicked at 'attempt to subtract with overflow', src/filters/mod.rs:218:22`

- [Rust implementation](https://github.com/cobalt-org/liquid-rust/blob/master/src/filters/mod.rs)
- [Ruby implementation](https://github.com/Shopify/liquid/blob/master/lib/liquid/standardfilters.rb)
- [Relevant tests](https://github.com/cobalt-org/liquid-rust/search?q=liquid+264&unscoped_q=liquid+264)
  - Once this is working, ideally the test will start failing.  We would just need to remove the `#[should_panic]` on it
  - If the test doesn't start failing, please temporarily remove `#[should_panic]`, and find or create an issue for the new failure

### PR DESCRIPTION
**Pull request by Geobert**
Fixes #264

### CODE REVIEW

**Comment by Geobert** [comment]
> error Subject should not be punctuated but found ``` 

I found this one a bit too much, isn't it?

File: crates/lib/src/stdlib/filters/string/truncate.rs [hunk 1]
```diff
@@ -72,13 +72,15 @@ impl Filter for TruncateFilter {
         let args = self.args.evaluate(runtime)?;
 
         let lenght = args.lenght.unwrap_or(50) as usize;
-
         let truncate_string = args.ellipsis.unwrap_or_else(|| "...".into());
-
-        let l = cmp::max(lenght - truncate_string.len(), 0);
+        let diff = if lenght >= truncate_string.len() {
```
Comment (epage) [inline crates/lib/src/stdlib/filters/string/truncate.rs:76]: Oh `lenght`.  This is why I want to get `typos` done so I can run it in CI :)

**Comment by epage** [comment]
> > error Subject should not be punctuated but found ```
> 
> I found this one a bit too much, isn't it?

https://github.com/crate-ci/committed/issues/62

File: crates/lib/src/stdlib/filters/string/truncate.rs (hunk 1 above)
Comment (Geobert) [reply crates/lib/src/stdlib/filters/string/truncate.rs:76]: Oh, didn't see it was wrong… #french :p I'll fix it :)

**Comment by epage** [comment]
Thanks!

**Pull request merged**